### ISSUE TITLE
Error while trying to bundle a Bootstrap JS

### ISSUE THREAD

**Issue by shelland** [failwyn/BundlerMinifier#2]
### Installed product versions
- Visual Studio: 2022
- This extension: latest from the gallery

### Description
Trying to minify (or bundle a minified file) a bootstrap library (bootstrap.bundle.js). Adding this file to bundle causes a NullReferenceException error in the VS errors list. Tried to bundle the most recent version of Bootstrap 5.

Seems like this issue is caused by NUglify library. Current NUglify version is 1.16.4, bundled with this extension: 1.13.8. As I see this issue was fixed in some recent versions of that library.

### Steps to recreate
1. Add a bootstrap bundle file to project
2. Try to bundle it by adding to `bundleconfig.js`

### Current behavior
Throws an error

### Expected behavior
Success

**Comment by failwyn** [comment]
Do you have a project that I can reproduce this with for testing? Also, can you link the issue that was fixed in NUglify?

**Comment by shelland** [comment]
[WebApplication2Bundler.zip](https://github.com/failwyn/BundlerMinifier/files/7739141/WebApplication2Bundler.zip)
I've attached a sample empty project with Bootstrap library and bundler config file. I'm not sure about Nuglify issue, there were some discussion somewhere that it should be fixed but I couldn't find it.

**Comment by failwyn** [comment]
This appears to be an issue with NUglify, I updated to the latest version and it still gives an error for line 1567 of the bootstrap file in your sample application.

The following code triggers the "Strict-mode does not allow duplicate property names" error
```
        const config = {
            ...Manipulator.getDataAttributes(target),
            ...Manipulator.getDataAttributes(this)
          }; 
```

**Comment by failwyn** [comment]
Fix available in version 2.9.3
